api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -577,6 +577,8 @@ message ScopedRoutes {
     }
 
     // The final scope key consists of the ordered union of these fragments.
+    // Fragments of the built scope key are compared in order with the fragments of a ScopedRouteConfiguration.
+    // A missing fragment will make the key a "NULL", i.e., the computed key doesn't match any key.
```

Comment:
Consider using "invalid" instead of "NULL"? Also, it may be clearer to say "A missing fragment during comparison".

Reply:
done